src/mainwindow.cpp

```diff
@@ -9400,6 +9400,12 @@ void MainWindow::openNotesContextMenu(
     }
 
     QAction *removeAction = noteMenu.addAction(tr("&Remove notes"));
+
+    QAction *moveToThisSubFolderAction = nullptr;
+    if (ui->noteTreeWidget->selectedItems().count() == 1) {
+        moveToThisSubFolderAction = noteMenu.addAction(tr("Go to this note's folder"));
```

Comment:
Can you please call it like "Jump to the note's sub-folder". And you need to check if sub-folders are activated for the current note folder. (there is a method hanging around somewhere that does that)

Reply:
> Can you please call it like "Jump to the note's sub-folder".

Sure. I was unsure about the label

And you need to check if sub-folders are activated for the current note folder. (there is a method hanging around somewhere that does that)

Will do!

Reply:
Awesome!